stylesheets/Components/nav.css

```diff
@@ -0,0 +1,22 @@
+.nav-header{
+    display: flex;
+    flex-direction: row;
```

Comment:
not required to explicitly specify flex direction when it is row and you dont have a media query that sets it to column.

Reply:
okay